docs/Xamarin.Forms.Core/Xamarin.Forms/Slider.xml

```diff
@@ -182,8 +182,7 @@ namespace FormsGallery
       <Docs>
         <summary>Gets or sets the maximum selectable value for the Slider. This is a bindable property.</summary>
         <value>A double.</value>
-        <remarks>
-        </remarks>
+        <remarks>Must be data-bound before Value to avoid Value being set to default Maximum if Value is greater than default Maximum.</remarks>
```

Comment:
What if we did something like this?

Coerce function for `Value`

```
var slider = (Slider)bindable;

if (!slider._isValueSet)
{
	if (slider.Value > slider.Maximum)
	slider.Maximum = slider.Value;

	if (slider.Value < slider.Minimum)
	slider.Minimum = slider.Value;

	slider._isValueSet = true;
}

return ((double)value).Clamp(slider.Minimum, slider.Maximum);
```

The behavioral change here would be that Minimum and Maximum values would be readjusted upon initial set of Value. In this case, the documentation changes can be removed (or edited to explain this change).